src/azure-cli/requirements.txt

```diff
@@ -0,0 +1,167 @@
+-i https://pypi.org/simple
```

Comment:
I guess this file is generated from the lock?

Reply:
Indeed!